mne/beamformer/_lcmv.py

```diff
@@ -253,6 +253,8 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
                 tmp = np.dot(Gk.T, np.dot(Cm_inv_sq, Gk))
                 eig_vals, eig_vecs = linalg.eig(np.dot(linalg.pinv(tmp),
                                                        np.dot(Wk, Gk)))
+                eig_vals = np.real(eig_vals)
+                eig_vecs = np.real(eig_vecs)
```

Comment:
As a reference for the orientation selection in unit-noise-gain beamformers, there is Sekihara & Nagarajan 2008.